tasko/loop.py

```diff
@@ -99,7 +99,7 @@ def resume():
         self._current = None
         return _yield_once(), resume
 
-    def schedule(self, hz: float, coroutine_function, *args, **kwargs):
+    def schedule(self, hz: float, coroutine_function, immediate=True, *args, **kwargs):
```

Comment:
This creates an ambiguous situation for anyone who is using this function with a schedule() invocation consisting only of kwargs arguments where one of the arguments is named `immediate`. Also it results in a toll paid every iteration on every coroutine to support the one-off requirement of bootstrapping a run-later function. Several ways to resolve this, but a simple example that follows your pattern is to instead add a function `schedule_later` which wraps the `coroutine_function` in the boilerplate
```
ran_once = False
async def fn():
  if !ran_once:
    await self._yield_once()
    ran_once = True
  coroutine_function(*args, **kwargs)
```
☝️ note: untested. Also, add a test if you're adding functionality 😄

Reply:
Thanks for the feedback, just added the new API with a test.